### PR TITLE
[Improvement](materialized-view) support multiple agg function have same base table slot

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes
support multiple agg function have same base table slot
## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindExpression.java [hunk 1]
```diff
@@ -806,7 +806,7 @@ private Plan bindSortWithoutSetOperation(MatchingContext<LogicalSort<Plan>> ctx)
     /**
      * For the columns whose output exists in grouping sets, they need to be assigned as nullable.
      */
-    private List<NamedExpression> adjustNullableForRepeat(
+    public static List<NamedExpression> adjustNullableForRepeat(
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindExpression.java:809]: better to move the method into PlanUtils.java, if it's used in two rules

File: fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/mv/SelectMaterializedIndexWithAggregate.java [hunk 2]
```diff
@@ -1087,7 +1084,7 @@ private static class ExprRewriteMap {
         /**
          * Replace map for expressions in project.
          */
-        public final Map<Expression, Expression> projectExprMap;
+        public final Map<Expression, List<Expression>> projectExprMap;
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/mv/SelectMaterializedIndexWithAggregate.java:1087]: please add a sample to explain the problem of multiple aggs with same slot, and how to solve it in this pr

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**